src/applications/financial-status-report/wizard/pages/Appeals.jsx

```diff
@@ -12,62 +13,64 @@ const Appeals = () => {
   }, []);
 
   return (
-    <div className="vads-u-background-color--gray-lightest vads-u-padding--2 vads-u-margin-top--2">
-      <p className="vads-u-margin-top--0">
-        Based on the information you provided, this isn’t the form you need.
-      </p>
-      <p className="vads-u-margin-bottom--0">
-        <strong>
-          If you disagree with our decision on your waiver request,
-        </strong>{' '}
-        you can{' '}
-        <a
-          href="/decision-reviews/board-appeal/"
-          onClick={() => {
-            recordEvent({
-              event: 'howToWizard-alert-link-click',
-              'howToWizard-alert-link-click-label': 'request a Board Appeal',
-            });
-          }}
-        >
-          request a Board Appeal
-        </a>
-        . When you choose this option, you appeal to a Veterans Law Judge at the
-        Board of Veterans' Appeals in Washington, D.C. A judge who’s an expert
-        in Veterans law will review your case.
-      </p>
-      <p className="vads-u-margin-top--1">
-        <a
-          href="/decision-reviews/board-appeal/"
-          onClick={() => {
-            recordEvent({
-              event: 'howToWizard-alert-link-click',
-              'howToWizard-alert-link-click-label':
-                'Find out how to request a Board Appeal',
-            });
-          }}
-        >
-          Find out how to request a Board Appeal
-        </a>
-      </p>
-      <p>
-        <strong>Note: </strong>
-        You have one year from the date on your decision letter to request a
-        Board Appeal, unless you have a{' '}
-        <a
-          href="/decision-reviews/contested-claims"
-          onClick={() => {
-            recordEvent({
-              event: 'howToWizard-alert-link-click',
-              'howToWizard-alert-link-click-label': 'contested claim',
-            });
-          }}
-        >
-          contested claim
-        </a>
-        .
-      </p>
-    </div>
+    <DelayedLiveRegion>
+      <div className="vads-u-background-color--gray-lightest vads-u-padding--2 vads-u-margin-top--2">
+        <p className="vads-u-margin-top--0">
```

Comment:
I'd recommend bumping this `p` tag up to a `h2` since it is a key result on the page ✌️ 
Most screen reader users (70% ish) navigate using headings on the page, so it would make this results section even more discoverable. I'd also recommend giving it a unique id e.g. `<h2 id="results">` so we can link it to the `<section>` with `aria-labelledby`

Reply:
Implemented  🙂